src/coord/rmr/rq.c

```diff
@@ -124,7 +123,9 @@ static void verify_uv_thread() {
     uv_timer_init(uv_default_loop(), &topologyFailureTimer);
     uv_async_init(uv_default_loop(), &topologyAsync, topologyAsyncCB);
     // Verify that we are running on the event loop thread
-    RedisModule_Assert(uv_thread_create(&loop_th, sideThread, NULL) == 0);
+    int uv_thread_create_status = uv_thread_create(&loop_th, sideThread, NULL);
+    RedisModule_Assert(uv_thread_create_status == 0);
+    REDISMODULE_NOT_USED(uv_thread_create_status);
```

Comment:
Was that required? `RedisModule_Assert` is running on release build right?

Reply:
Correct. But this is a better practice